Makefile.am

```diff
@@ -1371,12 +1371,18 @@ network_la_LDFLAGS += $(GCRYPT_LDFLAGS)
 network_la_LIBADD += $(GCRYPT_LIBS)
 endif
 
-test_plugin_network_SOURCES = src/network_test.c \
-			      src/daemon/configfile.c \
-			      src/daemon/types_list.c
-test_plugin_network_CPPFLAGS = $(AM_CPPFLAGS)
-test_plugin_network_LDFLAGS = $(PLUGIN_LDFLAGS)
-test_plugin_network_LDADD = liboconfig.la libplugin_mock.la libmetadata.la
+test_plugin_network_SOURCES = \
+	src/network_test.c \
+	src/utils_fbhash.c \
+	src/daemon/configfile.c \
+	src/daemon/types_list.c
+test_plugin_network_LDFLAGS = $(PLUGIN_LDFLAGS) $(GCRYPT_LDFLAGS)
```

Comment:
You'll need `test_plugin_network_CPPFLAGS = $(AM_CPPFLAGS) $(GCRYPT_CPPFLAGS)` as well I think?

Reply:
Good catch, thanks!